sydent/sydent.py

```diff
@@ -174,6 +175,18 @@ def set_from_comma_sep_string(rawstr):
         # The number of characters from the beginning to reveal of the email's domain
         # portion (right of the '@' sign)
         'email.third_party_invite_domain_obfuscate_characters': '3',
+
+        # Adds an extra layer of obfuscation, ensuring that even in the case of a username, domain
+        # or component containing very few characters - the entire string will not be shown.
+        #
+        # The algorithm works like so:
+        #   * If the string's length is greater than the cutoff value specified
+        #     by the above options, stop. Otherwise,
+        #   * If the string's length > 5, obfuscate to 3 characters.
+        #   * If the string's length > 1, obfuscate to 1 character
```

Comment:
Do we care about saying what an empty string does here?

Reply:
I think it may be worth mentioning, though I don't believe either side of the `@` in an email address can be empty.